recipe/meta.yaml

```diff
@@ -48,21 +45,18 @@ build:
 
 requirements:
   build:
+    - {{ stdlib('c') }} # https://github.com/conda-forge/conda-forge.github.io/issues/2102
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
     - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - sysroot_{{ target_platform }} 2.17  # [linux]
-    # TODO: Update aarch64 to use sysroot 2.28 when compatible CI images are
-    # available on conda-forge
-    #- sysroot_{{ target_platform }} 2.28  # [aarch64]
   host:
     - cuda-version 11.0  # [(cuda_compiler_version or "").startswith("11")]
     - cuda-version 12.0  # [(cuda_compiler_version or "").startswith("12")]
     - patchelf >=0.12  # [linux]
     # to suppress ".so not found" errors
-    - libzlib {{ zlib }}       # [linux]
-    - libzlib-wapi {{ zlib }}  # [win64]
+    - libzlib 1.2       # [linux]
```

Comment:
why were these zlib put in place. this kinda goes against conda-forge's best practices.

why not just have

```
- zlib
```

Reply:
Thanks @hmaarrfk for pointing this out. This was done when I didn't have greater understanding of conda-forge (and I am still learning!). I had noticed the global pinning had been changed from 1.2 to 1, so I wrongly thought it was a downgrade, and so I wanted to stick with 1.2. Now I understand that 1 means 1.* so it covers 1.3.1 which is the latest and great version as of now. Going forward, I will stick w/ the previous {{ zlib }} to depend on global pinning. If there is no strong objection I am inclined to leave it as-is for cudnn 9.2.1 ...

Reply:
ok my main concern was about the runtime requirement, but it seems to cover 

1.3.1
![image](https://github.com/user-attachments/assets/08c311b7-128d-4aec-a759-604c0e8736a7)